modules/placement-driver-api/src/main/java/org/apache/ignite/internal/placementdriver/event/PrimaryReplicaEvent.java

```diff
@@ -33,7 +33,7 @@ public enum PrimaryReplicaEvent implements Event {
      * <p>Notes:</p>
      * <ul>
      *     <li>This event will fire strictly after the completion of the future from {@link PlacementDriver#awaitPrimaryReplica}.</li>
-     *     <li>This event will fire when on node recovery and will indicate the primary replica at the time the node was stopped.</li>
+     *     <li>This event will fire in the metastore thread.</li>
```

Comment:
So the old statement is not correct, right?

Reply:
Yep